README.asyncpg.md

```diff
@@ -9,9 +9,81 @@ There is a customized version of the FastAPI SQL database tutorial for
 
 https://github.com/gordthompson/fastapi-tutorial-cockroachdb-async
 
-### Database support for Alembic
+### Default transaction isolation level
 
-CockroachDB version 23.1 or later is required to work with Alembic.
+Applications using asyncpg that were developed prior to CockroachDB's inclusion of
+READ COMMITTED transaction isolation may operate on the assumption that the default
+isolation level will be SERIALIZABLE. For example, 
+
+```python
+import asyncio
+
+from sqlalchemy.ext.asyncio import create_async_engine
+
+
+async def async_main():
+    engine = create_async_engine(
+        "cockroachdb+asyncpg://root@localhost:26257/defaultdb",
+    )
+    async with engine.begin() as conn:
+        result = await conn.exec_driver_sql("select version()")
+        print(result.scalar().split("(")[0])  # CockroachDB CCL v23.2.4
+        
+        result = await conn.exec_driver_sql("show transaction isolation level")
+        print(result.scalar())  # serializable
+
+
+asyncio.run(async_main())
+```
+
+With current versions of CockroachDB, the default transaction isolation level
+**for asyncpg only** is now READ COMMITTED
```

Comment:
@gordthompson We want to keep Serializable as the default. While we support read committed now, CockroachDB continues to default to read committed and we want to keep this behavior consistent across all of our tooling.

Reply:
The fix for this will be included in SQLAlchemy 2.0.38. I will update this PR once the new version is released.